src/pages/Admin/Venue/VenueDeleteModal.tsx

```diff
@@ -1,106 +1,64 @@
+  const [{ error, loading: isDeleting }, deleteVenue] = useAsyncFn(async () => {
+    await firebase.functions().httpsCallable("venue-deleteVenue")({
+      id: venueId,
+    });
+    onDelete?.();
+  }, [onDelete, venueId]);
 
   return (
-    <Modal show={show} onHide={closeDeleteModal}>
-      <div className="venue-delete-modal-container">
-        <h2 className="centered">Delete venue</h2>
+    <Modal show={show} onHide={onHide}>
+      <div className="VenueDeleteModal">
+        <h2 className="centered">Delete space</h2>
         <div className="secondary-action">
-          WARNING: Permanently removes a venue from SparkleVerse
+          WARNING: This action cannot be undone and will permanently remove the
+          space <b>{venueName}</b>!
         </div>
-        {!deleted && (
-          <>
-            <div className="input-group">
-              <span className="info">
-                WARNING: This action cannot be undone! Are you sure you wish to
-                delete {venue.name}?
-              </span>
+        {isDeleting && (
+          <div className="centered-flex">
+            <div className="spinner-border">
+              <span className="sr-only">Loading...</span>
             </div>
-            {deleting && (
-              <div className="centered-flex" style={{ marginBottom: 10 }}>
-                <div className="spinner-border">
-                  <span className="sr-only">Loading...</span>
-                </div>
-              </div>
-            )}
-            <div className="input-group">
-              <button
-                disabled={deleting}
-                className="btn btn-danger btn-block btn-centered"
-                onClick={deleteVenue}
-              >
-                Yes, Delete
-              </button>
-              {error && <span className="input-error">{error}</span>}
-              <button
-                disabled={deleting}
-                className="btn btn-primary btn-block btn-centered"
-                onClick={closeDeleteModal}
-              >
-                No, Cancel
-              </button>
-            </div>
-          </>
-        )}
-        {deleted && (
-          <div className="input-group">
-            <span className="info">Venue has been permanently deleted.</span>
-            <button
-              className="btn btn-primary btn-block btn-centered"
-              type="submit"
-              onClick={closeDeleteModal}
-            >
-              Close
-            </button>
           </div>
         )}
+        <div className="VenueDeleteModal__buttons">
+          <ButtonNG
+            disabled={isDeleting}
+            variant="danger"
+            onClick={deleteVenue}
+          >
+            Yes, Delete
+          </ButtonNG>
+          <ButtonNG disabled={isDeleting} variant="primary" onClick={onCancel}>
+            No, Cancel
+          </ButtonNG>
+        </div>
+        {error && <span className="input-error">{error}</span>}
```

Comment:
Could use something like https://github.com/sparkletown/sparkle/tree/fix/create-venue/src/components/atoms/AutoSpan to streamline display of simple error messages.

Example:
```suggestion
        <AutoSpan variant="error">{error}</AutoSpan>
```

Reply:
We actually have this in a reusable component `InputField` but sadly this is not a form and we can't use it. Another option is `ErrorMessage` from `react-hook-form`

Reply:
@denisdimitrov whichever you think best eliminates the need of `input-error` class which eventually might help in reducing the legacy baggage of `globals.scss`

Reply:
This is old legacy component and I don't want to waste time and resources on this. There are probably 50 instances of this className and it would be much easier to fix it in a single PR

Reply:
Well, a single PR is one way to go about it, provided it doesn't require widely differing approaches to replacing it and it doesn't cause merge issues to large number of PRs touching any of those 50 places.